ccd-definition/FixedLists/CMONextHearingType.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "CMONextHearingType",
+    "ListElementCode": "FURTHER_CASE_MANAGEMENT",
```

Comment:
no hearing because of listElementCode character limit? 

Would FURTHER_CASE_MGMT_HEARING work?